.github/workflows/integration-build-product.yml

```diff
@@ -117,8 +117,28 @@ env:
   PROJECT_DATA_VIEWER_API: "https://personal-7vf0v2cu.outsystemscloud.com/ProjectDataViewer5/rest/V1//CreateProject"
   MINIO_S3_URL: "https://s3.mov.ai"
   MINIO_PUBLIC_URL: "https://minio.mov.ai"
+  TIMEOUT: 720
```

Comment:
is 720 still a good default value then ? 12h seems to be too much if for example we do a 5h run ?

Reply:
valid